### PR TITLE
Api feature

### PR DESCRIPTION
**Pull request by Armynerh**
Used the  external API: [API Ninjas](https://api-ninjas.com/api/quotes)
Created a new component for displaying a quote
Fetched data inside of the new component
Showed a loading state
Showed an error state
Imported the new component into the main component (next to the Calculator)
It is on the same page as the Calculator

### CODE REVIEW

File: src/components/Quote.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  if (error) {
+    return <p>{error}</p>;
+  }
+
+  return (
+    <div>
+      <h1>Quote of the Day</h1>
+      <p><em>{quote}</em></p>
+    </div>
```
Comment (Cabraham1) [inline src/components/Quote.js]: - [x] Nice work so far 👍🏿 , but as per project requirement  you are to use the map method  to display the quotes 
 
like this 
 <ul>
      {quotes.map(({ author, quote }) => (
        <li key={author}>{quote}</li>
      ))}
    </ul>

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Armynerh   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Armynerh** [comment]
@Cabraham1  Could you kindly point out to me the part of the project requirement that states that the map method must be used? It seems I missed it.

File: src/components/Quote.js (hunk 1 above)
Comment (Cabraham1) [reply src/components/Quote.js]: here it is as code review guidelines  @Armynerh 

Screenshot 2023-08-18 at 4 37 06 PM

Comment (Armynerh) [reply src/components/Quote.js]: Thank you, @Cabraham1, for providing more clarity. I never knew we were supposed to display the author as well. I appreciate your feedback.

**Review by mckent05 — APPROVED** [review]
Hi @Armynerh, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job fetching quotes from the given API 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**